foodsaving/groups/tasks.py

```diff
@@ -22,6 +29,34 @@ def record_group_stats():
     write_points(points)
 
 
+@db_periodic_task(crontab(hour=2, minute=0))  # 2am every day
+def process_inactive_users():
+    now = timezone.now()
+
+    count_users_flagged_inactive = 0
+    count_users_removed = 0
+
+    remove_threshold_date = now - timedelta(days=settings.NUMBER_OF_DAYS_UNTIL_REMOVED_FROM_GROUP)
+    for gm in GroupMembership.objects.all().filter(lastseen_at__lte=remove_threshold_date, active=False):
+        email = prepare_user_removed_from_group_email(gm.user, gm.group)
+        email.send()
+        gm.delete()
+        count_users_removed += 1
+
+    inactive_threshold_date = now - timedelta(days=settings.NUMBER_OF_DAYS_UNTIL_INACTIVE_IN_GROUP)
+    for gm in GroupMembership.objects.all().filter(lastseen_at__lte=inactive_threshold_date, active=True):
```

Comment:
Ah, you don't actually need the call to `all()` here and above. Makes the line a bit shorter.